osu.Game.Rulesets.Osu/Mods/InputBlockingMod.cs

```diff
@@ -24,6 +26,10 @@ public abstract partial class InputBlockingMod : Mod, IApplicableToDrawableRules
         public override double ScoreMultiplier => 1.0;
         public override Type[] IncompatibleMods => new[] { typeof(ModAutoplay), typeof(ModRelax), typeof(OsuModCinema) };
         public override ModType Type => ModType.Conversion;
+        public override bool Ranked => IndicationOnly.Value;
```

Comment:
```suggestion
        public override bool Ranked => UsesDefaultConfiguration;
```

Reply:
Please correct me if I am wrong here. I thought that the default configuration of the InputBlocking mods is intended to be unranked. With the original PR, only the IndicationOnly version of the mod is made ranked, because it is essentially nomod, while the actually input-blocking version with default configuration is left unranked. This change would make it so that the default configuration version is ranked, while the indication-only version isn't. From my understanding, this is not consistent with how the mods behave at the moment, and Ranked should either be left as `IndicationOnly.Value` or changed to `!UsesDefaultConfiguration` to follow the convention. Alternatively, it can be changed to a constant `true` (which would be outside the scope of the original PR) or left as a constant `false`, which now I think is the best way to proceed.

Reply:
I don't get what you're trying to say, but maybe you should apply the diff and test before saying anything.